packages/rum-core/package.json

```diff
@@ -4,25 +4,16 @@
   "description": "Elastic apm core",
   "license": "MIT",
   "main": "src/index.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/elastic/apm-agent-js-core.git"
-  },
+  "repository": "https://github.com/elastic/apm-agent-js-base/tree/master/packages/rum-core",
   "scripts": {
-    "prepare-patch": "sh ../../dev-utils/prepare-release patch && npm run push-release",
-    "prepare-minor": "sh ../../dev-utils/prepare-release minor && npm run push-release",
-    "prepare-major": "sh ../../dev-utils/prepare-release major && npm run push-release",
-    "push-release": "git push origin master --follow-tags && npm publish",
-    "changelog": "node node_modules/conventional-changelog-cli/cli.js  -p angular -i CHANGELOG.md -s",
     "karma": "karma start",
     "karma-bench": "karma start karma.bench.conf.js",
     "karma-coverage": "karma start --coverage",
     "runUnitTests": "node ../../dev-utils/run-script.js runUnitTests true packages/rum-core",
     "test": "npm run runUnitTests"
   },
   "files": [
-    "src/**/*.js",
-    "test/**/*.js"
```

Comment:
There's a reference to `elastic-apm-js-core/test` in `test/e2e.js` in the js-base. would you please either refactor that one or revert this change?

Reply:
that shouldn't be a problem, since we are in mono repo this should just work without any problem.. I just removed it from npm publishing part.

Reply:
Yeah, but let's not have invalid reference!